content/release-2024-0/streaming-analytics-2024-0-bundle/2024_0.md

```diff
@@ -13,3 +13,11 @@ See also [Developing apps with the Streaming Analytics application](https://cumu
 in the *Streaming Analytics* guide.
 
 For more detailed information, see [Receiving update notifications](https://documentation.softwareag.com/pam/10.15.3/en/webhelp/pam-webhelp/index.html#page/pam-webhelp%2Fco-ConApaAppToExtCom_cumulocity_receiving_update_notifications.html) in the Apama documentation for the Cumulocity IoT transport connectivity plug-in.
+
+### Improvements in Analytics Builder
+
+The **Type** parameter of the [Constant Value](https://cumulocity.com/guides/10.19.0/streaming-analytics/block-reference/#constant-value) block
+now supports float and boolean value types, in addition to the existing string and JSON types.
+The **Type** parameter is now optional. If a type is not selected, the type of the output value is set based on the type of the input parameter value.
+<!-- The following is not mentioned in the block reference - so why mention it here??? Is this important for the end user? -->
```

Comment:
Yeah I don't think we need to mention this.

Reply:
Thanks, removed it.